src/tests/ftest/util/java_utils.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+from apricot import TestWithServers
+from daos_utils import DaosCommand
+from test_utils_pool import TestPool
+from test_utils_container import TestContainer
+from general_utils import run_pcmd
+
+class JavaUtils(TestWithServers):
+    """
+        Base class for Java test code
+        :avocado: recursive
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Initialize a JavaUtils object."""
+        super().__init__(*args, **kwargs)
+        self.pool = None
+        self.container = None
+        self.jdir = None
+
+    def _create_pool(self):
+        """Create a TestPool object.
+        """
+        # Get the pool params
+        pool = TestPool(
+            self.context, dmg_command=self.get_dmg_command())
+        pool.get_params(self)
+        # Create a pool
+        pool.create()
+        return pool
+
+    def _create_cont(self, pool, path=None):
+        """Create a TestContainer object to be used to create container.
+           Args:
+               pool (TestPool): pool object
+               path (str): Unified namespace path for container
+        """
+        # Get container params
+        container = TestContainer(pool, daos_command=DaosCommand(self.bin))
+        container.get_params(self)
+        if path is not None:
+            container.path.update(path)
+        # create container
+        container.create()
+        return container
+
+    def java_version(self):
+        """Check if java is installed.
+
+        Returns:
+            bool: whether java is installed or not.
+
+        """
+        # look for java home
+        command = [u"source {}/daos-java/find_java_home.sh".format(self.jdir)]
+        return self.run_command(self.hostlist_clients, command)
+
+    def run_command(self, hosts, commands, verbose=True, timeout=60):
```

Comment:
(pylint-inconsistent-return-statements) Either all return statements in a function should return an expression, or none of them should.